Server/controllers/authController.js

```
@@ -47,56 +46,61 @@
-            });
-
-        return res.status(200).json({
-            success: true, msg: successMessages.AUTH_CREATE_USER, data: {user: newUser, token: token},
+    const newUser = await req.db.insertUser({ ...req.body }, req.file);
+    logger.info(successMessages.AUTH_CREATE_USER, {
+      service: SERVICE_NAME,
+      userId: newUser._id,
+    });
+
+    const userForToken = { ...newUser._doc };
+    delete userForToken.profileImage;
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)